packages/config/rollup-configs.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable no-undef */
```

Comment:
is this file needed?

Reply:
haa, fighting linters is tricky, I'll see what I can do, the linter is unhappy because it's actually a node code, and not a browser, the `undef` stuff are node things like `process`  it's defined in node, but the linter is not happy.

if feeding the linter god makes things even more not nice, I'll just leave this comment.

Reply:
you mean the whole file ?

the file is the rollup configuration defaults used as base for other files, so all packages will have the same rollup base configuration and plugins.

the rollup configuration in types package use it as base template.

Reply:
I've meant the whole file. Adding rollup deserves a dedicated commit:)